js/hitbtc3.js

```diff
@@ -1173,14 +1222,36 @@ module.exports = class hitbtc3 extends Exchange {
         //         "volume_quote":"0.000018"
         //     }
         //
-        return [
-            this.parse8601 (this.safeString (ohlcv, 'timestamp')),
-            this.safeNumber (ohlcv, 'open'),
-            this.safeNumber (ohlcv, 'max'),
-            this.safeNumber (ohlcv, 'min'),
-            this.safeNumber (ohlcv, 'close'),
-            this.safeNumber (ohlcv, 'volume'),
-        ];
+        // Mark, Index and Premium Index
+        //
+        //     {
+        //         "timestamp": "2022-04-01T01:28:00.000Z",
+        //         "open": "45146.39",
+        //         "close": "45219.43",
+        //         "min": "45146.39",
+        //         "max": "45219.43"
+        //     },
+        //
+        const ohlcvLength = Object.keys (ohlcv).length;
+        if (ohlcvLength > 5) {
```

Comment:
You don't need to check this, `this.safeNumber (ohlcv, 'volume')` will return `undefined` if `volume` isn't present. This line is also causing the build to fail with `> not supported between instances of dict and int` for some reason, which makes no sense to me, but the good news is that you can just remove this line and the code inside the if